### ISSUE TITLE
"Session operation has timed out" on Azure Function

### ISSUE THREAD

**Issue by maikgroenewegen** [sshnet/SSH.NET#151]
I'm try to connect to a SFTP server from an Azure Function and I keep getting a "Session operation has timed out". I'm on version "2016.0.0". I've build a similar function using WinSCP and that works fine and the same applies to running the CLI command SFTP from the Kudu prompt. 

I'm using the following code:

public static void Run(TimerInfo myTimer, TraceWriter log)
{
            string sftp_host = <HOST>;
            string sftp_user = <USER>;
            string sftp_privateKey = @<LOCATION OF KEYFILE>
            var stream = new FileStream(sftp_privateKey, FileMode.Open, FileAccess.Read);
            var privateKeyFile = new PrivateKeyFile(stream);
            AuthenticationMethod authenticationMethod = new PrivateKeyAuthenticationMethod(<PASSPHRASE>, privateKeyFile);
            ConnectionInfo ConnNfo = new ConnectionInfo(sftp_host, sftp_user, authenticationMethod);
            SftpClient client = new SftpClient(ConnNfo);
            try
            {
                client.Connect();
                
            }
            catch (Exception ex)
                {
                    log.Info("Error Message: " + ex.Message);
                    client.Disconnect();
                }
}

**Comment by maikgroenewegen** [comment]
Also tried it in a PowerShell version (same error) and with the latest beta.

**Comment by drieseng** [comment]
I have access to Azure, but I have to admit that I never used it.
I'm afraid it'll take me ages to reproduce this issue.
This may be a duplicate of issue #130, but without more info it's hard to tell.

**Comment by maikgroenewegen** [comment]
I've you could let me know how to activate logging/debugging I might be able to get more detailed info.

**Comment by maikgroenewegen** [comment]
You're right it is related to issue #130.I implemented the fix as mentioned in the other issue and my PowerShell version now works.

**Comment by maikgroenewegen** [comment]
I changed line 109 in KeyExchangeDiffieHellman.cs to a fixed single number.
